ipaserver/dcerpc.py

```diff
@@ -58,7 +58,7 @@
 import pysss_nss_idmap
 import pysss
 import six
-from ipaplatform.paths import paths
+from ipaplatform.paths import paths  # pylint: disable=E0401
```

Comment:
For some reasons I do not understand, this import is not handled correctly by the pylint module extender.